src/styles/styles.js

```diff
@@ -2356,6 +2359,42 @@ const styles = {
         ...whiteSpace.noWrap,
     },
 
+    offlineFeedback: {
+        deleted: {
+            textDecorationLine: 'line-through',
+            textDecorationStyle: 'solid',
+        },
+        pending: {
+            opacity: 0.5,
+        },
+        error: {
+            flexDirection: 'row',
+            justifyContent: 'center',
+            alignItems: 'center',
+        },
+        container: {
+            ...spacing.pv2,
+        },
+        text: {
+            color: themeColors.textSupporting,
+            flex: 1,
+            textAlignVertical: 'center',
+        },
+        close: {
+            marginTop: 'auto',
+            marginBottom: 'auto',
```

Comment:
Not necessary, the close button is already being centered by the `alignItems: 'center'`

Reply:
Nice, yes, you are correct